### ISSUE TITLE
Add Brenda Stensland to /team page

### ISSUE THREAD

**Issue by natsteinmetz** [shescoding/shescoding-dot-org#120]

**Comment by AttackTheMak29** [comment]
@natsteinmetz - i'm working on this issue now. i wanted to check to verify that the company name is correct as Startucks in the first sentence. i was unable to find a public company by this name. let me know. :)

**Comment by cherchezlafemme** [comment]
@AttackTheMak29 you are correct, the company name is Starbucks, there is a typo in bio. Thank you for checking! You will also need a link to Brenda's linkedin to complete this task. The link is: https://www.linkedin.com/in/brendastensland/
Here is a link to a better quality image too: https://media.licdn.com/dms/image/C5603AQH-PKj_IZvQyQ/profile-displayphoto-shrink_800_800/0?e=1545868800&v=beta&t=bTOoHMW65dNtUHSrwbIt9Mc43xtiU7aRbVKu-DUoev4

**Comment by AttackTheMak29** [comment]
i've committed the changes. this is my first issue-specific commit, so let me know if i missed the correct way to do this. thank you!
